test/e2e/pull_test.go

```diff
@@ -127,12 +129,14 @@ var _ = Describe("Podman pull", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToString()).To(ContainSubstring("helloworld"))
+		Expect(session.ErrorToString()).To(ContainSubstring("The storage 'driver' option should be set in "))
+		Expect(session.ErrorToString()).To(ContainSubstring("A driver was picked automatically."))
```

Comment:
Do we want to keep this check, or would it make sense to write a driver into that `storage.conf` file?

Reply:
I'd keep for the sake of this PR, but if it should move, adding an issue to GitHub would make sense to me.

Reply:
I'd also keep it for now.